gems/aws-sdk-core/lib/aws-sdk-core/param_validator.rb

```diff
@@ -152,11 +156,11 @@ def shape(ref, value, errors, context)
       when MapShape then map(ref, value, errors, context)
       when DocumentShape then document(ref, value, errors, context)
       when StringShape
-        unless value.is_a?(String)
+        unless value.is_a?(String) || value.nil?
```

Comment:
I would avoid `unless` with more than one condition, I think that's also a rubocop recommendation.